openapi/components/responses/evidence-list.yaml

```diff
@@ -0,0 +1,241 @@
+                                System configurable in your Admin Lists Learning Moment area
+                              properties:
+                                id:
+                                  type: integer
+                                  description: the DB identifier of the Learning Moment Tags
+                                  example: 1
+                                name:
+                                  type: string
+                                  description: the name of the Learning Moment type, may not be unique
+                                  example: Class Work
+                            _links:
+                              type: object
+                              properties:
+                                update:
+                                  type: string
+                                  description: |
+                                    the link for the user to update the badge
+                                  example: /learning/badge/10/update
+                                delete:
+                                  type: string
+                                  description: |
+                                    the link for the user to delete the badge
+                                  example: /learning/badge/10
+                                achievementRecipients:
+                                  type: string
+                                  description: |
+                                    ???
+                                  example: /learning/badge/award/10/achievement/recipients
+                                goalRecipients:
+                                  type: string
+                                  description: |
+                                    ???
+                                  example: /learning/badge/award/10/goal/recipients
+                                achievementWithdraw:
+                                  type: string
+                                  description: |
+                                    ???
+                                  example: /learning/badge/award/10/achievement/withdraw
+                                goalWithdraw:
+                                  type: string
+                                  description: |
+                                    ???
+                                  example: /learning/badge/award/10/goal/withdraw
+                                icon:
+                                  type: string
+                                  description: |
+                                    The Badge Image, either system defined or selectable by the user\
+                                  example: /static/images/badges/sport/23.svg
+                    status:
+                      type: string
+                      example: goal
+                    isGoal:
+                      type: boolean
+                      example: true
+                    award:
```

Comment:
The `award` field is a reference to another badge award, so it'll share the same schema as this object.

(This field will be set for achievements which were awarded from a goal.)

Reply:
that makes sense... fixed